ep/build-velox/src/build_velox.sh

```diff
@@ -101,7 +101,7 @@ function compile {
   # Maybe there is some set option in velox setup script. Run set command again.
   set -exu
 
-  CXX_FLAGS='-Wno-error=stringop-overflow -Wno-error=cpp -Wno-missing-field-initializers -Wno-unknown-warning-option'
+  CXX_FLAGS='-Wno-error=stringop-overflow -Wno-error=cpp -Wno-missing-field-initializers -Wno-unknown-warning-option -Wno-template-id-cdtor'
```

Comment:
-Wno-template-id-cdtor seems to be gcc specific, not supported by clang.
```
clang-18 -Wno-template-id-cdtor a.cpp
warning: unknown warning option '-Wno-template-id-cdtor' [-Wunknown-warning-option]
```

Reply:
Sorry for confusion. Yes it is for GCC.

Reply:
What version of gcc? Could you post the relevant error? Thanks

Reply:
GCC 14

Reply:
It seems to be caused by C++20. Docker uses gcc11 and C++17 to compile, so this problem will not occur. You can use gcc11 or specify -std=c++17 to see if the problem can be solved.

Reply:
I see. Thanks. I am moving gluten build to a separate environment.